src/Mod/ConstraintSolver/App/G2D/ConstraintCurvePosPyImp.cpp

```diff
@@ -19,7 +19,7 @@ PyObject *ConstraintCurvePosPy::PyMake(struct _typeobject *, PyObject* args, PyO
             PyErr_SetString(PyExc_TypeError, "Only keyword arguments are supported");
             throw Py::Exception();
         }
-        HConstraintCurvePos p = (new ConstraintCurvePos)->self().downcast<ConstraintCurvePos>();
+        HConstraintCurvePos p = (new ConstraintCurvePos)->getHandle<ConstraintCurvePos>();
```

Comment:
wait a sec... It should have been changed just to `HConstraintCurvePos p = new ConstraintCurvePos;`.